saleor/graphql/tax/mutations/tax_configuration_update.py

```diff
@@ -149,13 +209,15 @@ def update_countries_configuration(cls, instance, countries_configuration):
             obj.charge_taxes = data["charge_taxes"]
             obj.display_gross_prices = data["display_gross_prices"]
             obj.tax_calculation_strategy = data.get("tax_calculation_strategy")
+            obj.tax_app_id = data.get("tax_app_id")
```

Comment:
Is it possible (should it be?) to clear the tax app and provide `null` as `taxAppId`?

Reply:
As for now I'm leaving an option to set `NULL`, just in case someone will want to turn off implemented behaviour.
Said field will become mandatory for Saleor 4.0.